HUD/GetMenuLayoutChangedEventDetails.md

```diff
@@ -0,0 +1,38 @@
+---
+ns: HUD
+aliases: ["0x7E17BE53E1AAABAF", "_GET_PAUSE_MENU_SELECTION_DATA"]
+---
+## GET_MENU_LAYOUT_CHANGED_EVENT_DETAILS
+
+```c
+// 0x7E17BE53E1AAABAF 0x6025AA2F
+void GET_MENU_LAYOUT_CHANGED_EVENT_DETAILS(int* oldMenuState, int* currentMenuState, int* selectedItemUniqueId);
+```
+
+Returns details of the current layout changed event. Also see the HasMenuLayoutChangedEventOccurred native.
+
+oldMenuState: This is the menuID of the last selected item minus 1000 (lastItem.menuID - 1000)
+currentMenuState: Same as oldMenuState except for the currently selected menu item
+selectedItemUniqueId: This is uniqueID of the currently selected menu item
+menu state values can be viewed here (or PauseMenuLUT.as in scaleform files): https://pastebin.com/ZiMpN3mN
+
+when the pausemenu is closed:
```

Comment:
This seems to be a lot of research documentation, it might be better to have this as a [Modding Tutorial](https://forum.cfx.re/c/development/tutorials/21)

Reply:
I've decided to keep this only because the previous description had it and didn't want to remove some information that could be deemed as useful, but if needed, i can remove it